docs/guide/personalization/developer_guide/tracking_api.md

```diff
@@ -197,49 +291,65 @@ The format of the URL is: 
 
 !!! note "Deprecated 'transfer' event"
 
-    Based on feedback of our customers and to make the purpose a bit clearer, we renamed the formerly named 'transfer' event to 'login'. As the name implies, it should be used when a user logs in. It covers exactly the same functionality as the transfer event. There is no need to change anything on the customer's side for existing implementations as the transfer event https://event.yoochoose.net/\[solutionid\]/\[customerid\]/transfer/\[sourceuserid\]/\[targetuserid\] will still be supported but should be considered as deprecated!
+    Based on feedback of our customers and to make the purpose a bit clearer, we renamed the formerly named 'transfer' event to 'login'. As the name implies, it should be used when a user logs in. It covers exactly the same functionality as the transfer event. There is no need to change anything on the customer's side for existing implementations because the transfer event https://event.yoochoose.net/\[solutionid\]/\[customerid\]/transfer/\[sourceuserid\]/\[targetuserid\] is still be supported, even though it should be considered as deprecated.
```

Comment:
To consider in future: when was the "former" name used? Is it still relevant for the current audience, that it, could they have encountered the old name at all?